File: dataset/human/python/file_7759.py
import std.stdio;

void main() {
	int n, k, s;
	scanf("%d %d %d\n", &n, &k, &s);
	if (s == 10^^9) {
		foreach(_;0..n-k) {
			write(1, " ");
		}
		foreach(_;0..k) {
			write(10^^9, " ");
		}
	}
	else {
		foreach(_;0..n-k) {
			write(s+1, " ");
		}
		foreach(_;0..k) {
			write(s, " ");
		}
	}
}

File: dataset/human/c/file_7759.c
#include <stdio.h>
 int main(void){
   int times,i;

   scanf("%d",&times);

   int K[times];
   int P[times];

   for(i=0;i<times;i++){
     scanf("%d %d",&K[i],&P[i]);
   }

   for(i=0;i<times;i++){
     if(K[i]%P[i]==0){
       printf("%d\n",P[i]);
     }else{
       printf("%d\n",K[i]%P[i]);
     }
   }
   return 0;
 }
